vault/resource_identity_entity_test.go

```diff
@@ -305,7 +305,7 @@ func TestReadEntity(t *testing.T) {
 			expectedRetries: 5,
 			wantError: fmt.Errorf(`failed reading %q`,
 				entity.JoinEntityID("retry-exhausted-custom-max-412")),
-			retryWait: time.Millisecond,
+			retryWait: 500 * time.Millisecond,
```

Comment:
This test is flaky sometimes so I am bumping this up to see if it will fix it. Not related to this PR at all but I couldn't help myself.

Reply:
Nice, things look a lot more robust under stress testing 👍🏼